.github/ISSUE_TEMPLATE/update-external-links-gallery.md

```diff
@@ -5,40 +5,42 @@ title: ''
 labels: 'external-links-gallery-submission'
 ---
 
-<!-- Please fill out the template below. -->
+<!-- Please fill out the template below by adding the necessary information in between the single quotes in each relevant or known field. -->
 
 Make sure the following items are specified :
 
-- `url`
-- `title`
-- `description`
+- [ ] `url`
+- [ ] `title`
+- [ ] `description`
+- [ ] `authors` (please credit the creators of the resource in at least one method (`name`, `affiliation`, `affiliation_url`, or `email`)
+- [ ] `tag` (at least one tag is required, this helps users access the link by searching via the relevant `packages`, `formats`, or `domains`)
 
 **Note**:
 
 - The thumbnail can be a url to a direct image link.
 
 ```yaml
 ---
-url: ''
-thumbnail: ''
-title: ''
-description: ''
-authors:
-  - name: ''
-    affiliation: ''
-    affiliation_url: ''
-    email: ''
-  - name: ''
+url: '' # The URL link to the resource you are submitting.
+thumbnail: '' # The URL link to the thumbnail image associated with this resource.
+title: '' # The title of the resource.
+description: '' # A brief description of the resource.
+authors: 
+  - name: '' # The name of the first author. Mulitiple authors are supported, add each author seperately.
+    affiliation: '' # The affiliation of the authors / first author (more appropriate than name if a large team worked on the resource).
+    affiliation_url: '' # The URL link to the affiliation website.
+    email: '' # The author's email, if publically available.
+  - name: '' # The name of the second+ author.
     affiliation: ''
     affiliation_url: ''
     email: ''
 tags:
   packages:
-    - unspecified
+    - unspecified # List each Python package associated with this resource on a seperate line (the dash is only necessary for the first entry).
   formats:
-    - unspecified
+    - unspecified # List the formats this resource takes (e.g. book, course, documentation, gallery, tutorial, video).
```

Comment:
Are their specific keywords that the user needs to choose from for formats, packages or domain? If so, maybe we should be more specific: e.g. you must choose from the following list.

Reply:
They can submit an entirely new keyword! But these are the keywords currently in play. I will specify this.